case_studies/san_leandro_high_school/01_club_fair/post_mortem.md

```diff
@@ -0,0 +1,25 @@
+# Post-mortem
+
+##### What We Did:
+  * Encouraged people to sign up.
+  * Passed out flyers.
+  * Explained our mission.
+
```

Comment:
Same comments for this section as in https://github.com/hackedu/hackedu/pull/206/files#r39449571.